pom.xml

```diff
@@ -349,6 +349,36 @@ under the License.
                     </transformers>
                 </configuration>
             </plugin>
+
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>xml-maven-plugin</artifactId>
+                <version>1.0.2</version>
+                <configuration>
+                    <indentSize>4</indentSize>
+                    <useDefaultFormatFileSet>false</useDefaultFormatFileSet>
+                    <formatFileSets>
+                        <formatFileSet>
+                            <directory>.</directory>
+                            <excludes>
+                                <exclude>**</exclude>
```

Comment:
This seems unnecessary to me, the include is sufficient.

Reply:
Hi @mbalassi, thank you for the feedback, I agree with your commend that include alone is sufficient. Have updated the PR, thank you

Reply:
Roger that. I will merge as soon as the first build returns green.

Reply:
Thank you!